### DISCUSSION TITLE
Issue with grammar file

### DISCUSSION THREAD

**Discussion by ankit289** [Sunbird-cQube/community#849] [General]
![image](https://github.com/Sunbird-cQube/community/assets/43002190/ad1abf6d-34a1-45a9-84c5-929a5793aaab)

The column progress id is dynamic in nature. Its neither a metric nor a dimension so can we handle it in grammar file.

**Comment by RNarmatha** [comment]
@dhanush-2397 - Could you please provide your guidence here ?

**Comment by dhanush-2397** [comment]
Hi,
The columns should be either metric or a dimension. It cannot have any values apart from this.
The current cQube architecture requires it be specified as either dimension or metric.
